lib/main_containers.pm

```diff
@@ -0,0 +1,71 @@
+# SUSE's openQA tests
+#
+# Copyright © 2021 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: module loader of container tests in SLES
+# Maintainer: qa-c@suse.de
+
+package main_containers;
+use base 'Exporter';
+use Exporter;
+use utils;
+use version_utils;
+use main_common qw(loadtest boot_hdd_image);
+use containers::utils qw(loadtest_containers);
+use testapi qw(check_var get_required_var get_var);
+use Utils::Architectures;
+use strict;
+use warnings;
+
+
+our @EXPORT = qw(
+  load_container_tests
+);
+
+sub load_container_tests {
+    my $runtime = get_required_var('CONTAINER_RUNTIME');
+    boot_hdd_image;
+
+    # Container Image tests
+    if (get_var('FLAVOR', '') =~ /Container-Image/) {
+        loadtest_containers 'host_configuration';
+        loadtest_containers 'docker_image';
+        loadtest_containers 'podman_image';
+    }
+
+    # Container Host tests
+    else {
+        if ($runtime eq 'docker') {
+            loadtest_containers 'docker';
+            unless (is_sle("<=15") and is_aarch64) {
+                loadtest_containers 'zypper_docker';
```

Comment:
This module should work everywhere.

Reply:
Not for this conditions, please check the table here: https://confluence.suse.com/display/ENGCTNRSTORY/Level+Test+Plan#LevelTestPlan-PackagesfromContainersModule

Reply:
I would then be interested why `zypper_docker` does not work on `aarch64` but I guess that OK for now.

Reply:
The package `zypper_docker` doesn't exist for aarch64 for <15 (12-SP3, 12-SP4, 12-SP5, 15). You can check that yourself, just go to scc.suse.com and search for `zypper_docker` package selecting the versions and arch.